app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
In my implementation you can have a direct access to a page anchor.
What would be the url format without an additional parameter?